### PR TITLE
Restructure hardware requirements table

### PR DESCRIPTION
**Pull request by jillian-maroket**
[This comment](https://github.com/harvester/docs/pull/543#issuecomment-2094242257) made me realize that the doc does not clearly distinguish between *requirements* and *recommendations*. Users typically want to know if the system can run on their existing hardware. This can prompt them to search for terms such as requirements/required and recommendations/recommended.

I understand that the team is more comfortable with testing/development vs. production. Because of this, I updated the existing hardware requirements table to make the distinction clearer. Markdown doesn't allow merging of table cells so some information is repeated.

Question: Do we want to add the following details that are mentioned [here](https://www.suse.com/suse-harvester/support-matrix/all-supported-versions/harvester-v1-3-x/)?
- "For best results, SUSE recommends using YES certified for SLES 15 SP3 or SP4 Hardware with Harvester: https://www.suse.com/yessearch/. Harvester is built on SLE technology and YES certified hardware has additional validation of driver and system board compatibility."
- "Only local disks or Hardware RAID is supported."
- "Testing environments with 1 GB NIC / 140 GB disk are not supported"

### CODE REVIEW

File: docs/install/requirements.md [hunk 1]
```diff
@@ -22,14 +22,15 @@ Certain versions of Harvester support the deployment of [single-node clusters](h
...
+| Hardware | Development/Testing | Production |
+| :--- | :--- | :--- |
+| CPU | x86_64 (with hardware-assisted virtualization); 8 cores minimum | x86_64 (with hardware-assisted virtualization); 16 cores minimum |
+| Memory | 32 GB minimum | 64 GB minimum |
+| Disk capacity	| 250 GB minimum (180 GB minimum when using multiple disks) | 500 GB minimum |
+| Disk performance | 5,000+ random IOPS per disk (SSD/NVMe); management node storage must meet [etcd]((https://www.suse.com/support/kb/doc/?id=000020100)) speed requirements | 5,000+ random IOPS per disk (SSD/NVMe); management node storage must meet [etcd]((https://www.suse.com/support/kb/doc/?id=000020100)) speed requirements |
+| Network card count | Management cluster network: 2 NICs; VM workload network: 2 additional NICs (does not apply to the [witness node](../advanced/witness.md)) | Management cluster network: 2 NICs; VM workload network: 2 additional NICs (does not apply to the [witness node](../advanced/witness.md)) |
```
Comment (bk201) [inline docs/install/requirements.md:31]: We can have a single nic for testing, @mingshuoqiu any comment on this?

Comment (w13915984028) [reply docs/install/requirements.md:31]: The current requirement is:

>Management cluster network: 2 NICs => 

Required: Management cluster network: 1 NIC
Recommend: 2+ NIC to form a reboust with capacity of backup/loadbalance 

>VM workload network:  

Optional, depends on user network planning, there maybe 0~N such networks
For each possibile `VM workload network`, 1 indepedent NIC is a basic requirentment; 2+ NICs to form a reboust bond ...

refer: https://docs.harvesterhci.io/v1.3/networking/index#cluster-network

Comment (jillian-maroket) [reply docs/install/requirements.md]: Thank you. I have updated the "network card count" information.

**Comment by jillian-maroket** [comment]
@bk201 I have updated the file. Please confirm that I can ignore the following items on the Support Matrix page.

- "For best results, SUSE recommends using YES certified for SLES 15 SP3 or SP4 Hardware with Harvester: https://www.suse.com/yessearch/. Harvester is built on SLE technology and YES certified hardware has additional validation of driver and system board compatibility."
- "Only local disks or Hardware RAID is supported."

Screenshot 2024-05-29 at 4 57 34 PM

**Comment by bk201** [comment]
> - "For best results, SUSE recommends using YES certified for SLES 15 SP3 or SP4 Hardware with Harvester: https://www.suse.com/yessearch/. Harvester is built on SLE technology and YES certified hardware has additional validation of driver and system board compatibility."
> - "Only local disks or Hardware RAID is supported."

@jillian-maroket Sorry for the delay. These sentences are important because we use a SLES derivative and don't support software RAID yet.

**Comment by jillian-maroket** [comment]
@bk201 Thank you for your comment. I have updated the doc to include the information from the Support Matrix page. PTAL

**Review by bk201 — APPROVED** [review]
Thanks!

**Pull request merged**